textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/Span.java

```diff
@@ -0,0 +1,19 @@
+package edu.uci.ics.textdb.common.field;
+
+public class Span {
+    private int start;

```

Comment:
OK. No need to change it then.  ("Short" may limit us on the length of documents.)